src/foam/nanos/crunch/ui/CapabilityWizardlet.js

```diff
@@ -8,14 +8,39 @@ foam.CLASS({
   name: 'CapabilityWizardlet',
   extends: 'foam.u2.wizard.BaseWizardlet',
 
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
   imports: [
-    'crunchController'
+    'crunchController',
+    'localeDAO'
   ],
 
   properties: [
     // Properties specific to CapabilityWizardSection
     {
-      name: 'capability'
+      name: 'capability',
+      postSet: function() {
+        var self = this;
```

Comment:
Do users ever set the title explicitly? If so, maybe we should do something like:
if ( this.title ) return;
So that we don't overwrite the explicitly specified capability.

Reply:
I don't think so, factory function is called but when displayed title is empty